README.md

```diff
@@ -1,3 +1,92 @@
 # SirMordred [![Build Status](https://travis-ci.org/grimoirelab/mordred.svg?branch=master)](https://travis-ci.org/grimoirelab/mordred)
 
-SirMordred is the tool used to coordinate the execution of the GrimoireLab platform.
+SirMordred is the tool used to coordinate the execution of the GrimoireLab platform, via two files: `setup.cfg` and `projects.json`.
+The former contains several sections to tune the execution of GrimoireLab, while the latter contains the URIs from where to extract
+the project data.
+
+## Setup.cfg
+
+This section goes through the different sections of the configuration file, commenting their attributes.
+
+### [es_collection]
```

Comment:
Hmmm, this documentation about the params is already included in the config.md file and generated automatically from the doc in the source code. I won't have it duplicated (is it synced in some way with the other file?).

Reply:
The PR is still not finished, I pushed it to see it how it is rendered in GitHub